packages/gatsby-cli/src/init-starter.js

```diff
@@ -67,8 +71,30 @@ const install = async rootPath => {
   process.chdir(rootPath)
 
   try {
-    let cmd = shouldUseYarn() ? spawn(`yarnpkg`) : spawn(`npm install`)
-    await cmd
+    const npmCmd = `npm install`
+    let response = npmCmd
+    if (shouldUseYarn()) {
+      const promptsAnswer = await prompts([
+        {
+          type: `select`,
+          name: `package_manager`,
+          message: `Which package manager would you like to use ?`,
+          choices: [
+            { title: `yarn`, value: `yarnpkg` },
+            { title: `npm`, value: npmCmd },
+          ],
+          initial: 0,
+        },
+      ])
+      response = promptsAnswer.package_manager
+    }
+    conf.set(`package_manager`, response)
+    if (response.includes(`yarn`)) {
+      await spawn(`rm package-lock.json`)
```

Comment:
Won't this fail on e.g., windows? 

I suppose we could just use the [`fs`](https://nodejs.org/api/fs.html#fs_fs_unlinksync_path) module to delete the file
```javascript
fs.unlinkSync(`package-lock.json`)
```